senza/components/taupage_auto_scaling_group.py

```diff
@@ -18,6 +18,11 @@
 from typing import Optional
 
 _AWS_FN_RE = re.compile(r"('[{]{2} (.*?) [}]{2}')", re.DOTALL)
+_CHANNEL_MAPPING = {
+    "stable": "LatestTaupageImage",
```

Comment:
Are the names `LatestTaupageImage`, `LatestTaupageStagingImage` and `LatestTaupageDevImage` used outside senza code? Why not just use `stable`, `staging` and `dev`?

Reply:
`LatestTaupageImage` is used written into the generated CF template, so it's possible that someone else depends on it. I don't think it's a good idea to break backwards compatibility over a trivial rename.